### PR TITLE
New layout of the Signup-init Screen

### PR DESCRIPTION
**Pull request by Aby-JS**

### CODE REVIEW

File: playground/react/src/assets/logo.svg
Comment (incorbador) [inline playground/react/src/assets/logo.svg:1]: what do we need this svg for?

File: packages/react/src/components/ui2/ErrorPopup.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const ErrorPopup = () => (
+  <div className='cb-error-popup-2'>
+    <div className='cb-error-popup-icon-2'>
+      <svg
+        xmlns='http://www.w3.org/2000/svg'
+        width='35'
+        height='29'
+        viewBox='0 0 35 29'
+      >
+        <path
```
Comment (incorbador) [inline packages/react/src/components/ui2/ErrorPopup.tsx:12]: Why do we define the svgs here inside the react package now? Previously we defined them in shared-ui and then reused them (this had the additional benefit that they were isolated in separate files where you can easily understand what one svg is about).

**Review by incorbador — COMMENTED** [review]
two small comments, otherwise good to merge.

File: packages/react/src/components/ui2/ErrorPopup.tsx (hunk 1 above)
Comment (Aby-JS) [reply packages/react/src/components/ui2/ErrorPopup.tsx:12]: will fix that.. it was for testing initially

File: playground/react/src/assets/logo.svg
Comment (incorbador) [reply playground/react/src/assets/logo.svg:1]: The conversation is resolved, the logo is still there though, why?

**Review by incorbador — CHANGES_REQUESTED** [review]
Take a look at the logo comment + the primary button should be in loading state after is has been clicked (until feedback from the backend is received).

Afterwards we can merge this PR.

**Pull request merged**